.travis.yml

```diff
@@ -9,12 +9,17 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=utils
+
 install:
-  - pip install -r requirements.txt
```

Comment:
If we do find a way to add back installing the items from requirements.txt, I think we'd want to do it in tox and not directly in .travis.yml.